linera-base/src/identifiers.rs

```diff
@@ -29,12 +32,30 @@ use crate::{
 
 /// An account owner.
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, WitLoad, WitStore, WitType)]
-#[cfg_attr(with_testing, derive(test_strategy::Arbitrary))]
 pub enum AccountOwner {
     /// Short addresses reserved for the protocol.
     Reserved(u8),
     /// 32-byte account address.
     Address32(CryptoHash),
+    /// 20-byte account EVM-compatible address.
+    Address20(Address),
```

Comment:
One possible debate here is whether we want to use `alloy_primitives::Address` or not. Potentially, this means we depend on `alloy` even when the `evm` feature is not set.

Reply:
Switched to `[u8; 20]`.